### ISSUE TITLE
Release fails on unpublished dottyJS

### ISSUE THREAD

**Issue by rossabaker** [Log4s/log4s#60]
This is the version we exclude from our build matrix.  It comes last, so the rest of the release can be done manually, but that's sloppy.

Maybe procrastination will work and dottyJS will be functional by next milestone.

```
[error] stack trace is suppressed; run last coreJS / publishSigned for the full output
[error] stack trace is suppressed; run last testingJS / publishSigned for the full output
[error] (coreJS / publishSigned) java.io.IOException: PUT operation failed because the destination file exists and overwriting is disabled:
[error]     source     : /home/ross/src/log4s/core/js/target/scala-2.13/log4s_sjs1_2.13-1.10.0-M1-sources.jar
[error]     destination: https://oss.sonatype.org/service/local/staging/deploy/maven2/org/log4s/log4s_sjs1_2.13/1.10.0-M1/log4s_sjs1_2.13-1.10.0-M1-sources.jar
[error] If you have a staging repository that has failed, drop it and start over.
[error] Otherwise fix the double publishing, or relax the setting as follows:
[error]     publishConfiguration := publishConfiguration.value.withOverwrite(true)
[error]     publishLocalConfiguration := publishLocalConfiguration.value.withOverwrite(true)
[error] 
[error] If you have a remote cache respoitory, you can enable overwriting as follows:
[error]     pushRemoteCacheConfiguration := pushRemoteCacheConfiguration.value.withOverwrite(true)
[error] (testingJS / publishSigned) java.io.IOException: PUT operation failed because the destination file exists and overwriting is disabled:
[error]     source     : /home/ross/src/log4s/testing/js/target/scala-2.13/log4s-testing_sjs1_2.13-1.10.0-M1.pom
[error]     destination: https://oss.sonatype.org/service/local/staging/deploy/maven2/org/log4s/log4s-testing_sjs1_2.13/1.10.0-M1/log4s-testing_sjs1_2.13-1.10.0-M1.pom
[error] If you have a staging repository that has failed, drop it and start over.
[error] Otherwise fix the double publishing, or relax the setting as follows:
[error]     publishConfiguration := publishConfiguration.value.withOverwrite(true)
[error]     publishLocalConfiguration := publishLocalConfiguration.value.withOverwrite(true)
[error] 
[error] If you have a remote cache respoitory, you can enable overwriting as follows:
[error]     pushRemoteCacheConfiguration := pushRemoteCacheConfiguration.value.withOverwrite(true)
```

**Comment by rossabaker** [comment]
This is a bigger issue now that we have two Dotty versions we're publishing against.  Need to come back and clean up #64.

**Comment by rossabaker** [comment]
Fixed by #75
